EMacco/abarian-backend
# #170468627 Delete address

#### What does this PR do?
This PR allows a user to be able to delete an address

#### Description of Task to be completed?
- user must be logged in
- user must be verified
- make sure that the address exists
- delete address

#### How should this be manually tested?
- checkout to this branch 
- send a delete request to `/addresses/:addressId`
- the address should be deleted

#### Any background context you want to provide?
- you must be logged in
- you must be verified
- address must exist

#### What are the relevant pivotal tracker stories?
[#170468627](https://www.pivotaltracker.com/story/show/170468627)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
